Add note for freshness stating the requirement to have both count and… (#7460)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/freshness.md

| Configuration | Description |
|--------------|-------------|
| `build_after` | Config nested under `freshness`. Used to determine whether a model should be rebuilt when new data is present, based on whether the specified count and period have passed since the model was last built. Although dbt checks for new data every time the job runs, `build_after` ensures the model is only rebuilt if enough time has passed and new data is available. |

> | `count` and `period` | Specify how often dbt should check for new data. For example, `count: 4, period: hour` means dbt will check every 4 hours.<br /><br /> Note that for every `freshness` config, it is required to either set values for both `count` and `period`, or set `freshness: null`.|

Reviewer: ```suggestion
| `count` and `period` | Specify how often dbt should check for new data. For example, `count: 4, period: hour` means dbt will check every 4 hours.<br /><br /> Note that for every `freshness` config, you're required to either set values for both `count` and `period`, or set `freshness: null`.|
```
